rest-server/src/models/user.js

```diff
@@ -224,24 +195,23 @@ const checkUserVc = (username, virtualCluster, callback) => {
   }
 };
 
-const getUserList = (next) => {
-  db.get(etcdConfig.storagePath(), {recursive: true}, (errMsg, res) => {
-    if (errMsg) {
-      next(new Error('UserListNotFound'), null);
-    } else {
-      const userInfoList = [];
-      res.forEach((value, key) => {
-        if (value === undefined && key !== etcdConfig.storagePath()) {
-          let userName = key.replace(etcdConfig.storagePath() + '/', '');
-          userInfoList.push({
-            username: userName,
-            admin: res.get(etcdConfig.userAdminPath(userName)),
-            virtualCluster: res.has(etcdConfig.userVirtualClusterPath(userName)) ? res.get(etcdConfig.userVirtualClusterPath(userName)) : 'default',
-          });
-        }
-      });
-      next(null, userInfoList);
+const getUserList = (callback) => {
```

Comment:
What is the purpose to change `next` to `callback`?

Reply:
In my understand of the coding style, when using `callback` as callback name, error will be the first argument, while using `next` as callback name, error will be the second argument (https://github.com/Microsoft/pai/blob/2e55ab6b9a48c4ac0767b82e7f904f6498a895c0/rest-server/src/models/vc.js#L76)

This `next` is the only exception and confuse me a while.

Reply:
Node.js [error document](https://nodejs.org/api/errors.html#errors_error_first_callbacks) and [util document](https://nodejs.org/api/util.html#util_util_callbackify_original) encourage "error-first callbacks".